aa-sdk/core/src/errors/client.ts

```diff
@@ -98,8 +100,6 @@ export class EntityIdOverrideError extends BaseError {
    * Initializes a new instance of the error message with a default message indicating that the nonce key is invalid.
    */
   constructor() {
-    super(
-      `Installing entityId of 0 overrides the owner's entity id in the account`
-    );
+    super(`EntityId of 0 is used by the owner and cannot be used`);
```

Comment:
what is the `entityId` of 0 used by the owner for?

Reply:
nit for wording: "EntityId of 0 is reserved for the owner" over "EntityId of 0 is used by the owner"